### PR TITLE
custom cloak search

### PR DESCRIPTION
**Pull request by aeiea**
no merge conflicts this time

### CODE REVIEW

File: js/settings.js [hunk 1]
```diff
@@ -116,8 +116,7 @@ function setFavicon(icon) {
   localStorage.setItem("tab", JSON.stringify(tabData));
 }
 
-function setCloak() { // applies only to premade cloaks
-  var cloak = document.getElementById("premadecloaks").value; // cloak seems kind of weird when you spell it out
```
Comment (DapperLS-Bot-Dev) [inline js/settings.js:120]: It wont work if you delete line 120 in settings.json. Can you add that line back?

**Review by DapperLS-Bot-Dev — CHANGES_REQUESTED** [review]

File: js/settings.js (hunk 1 above)
Comment (DapperLS-Bot-Dev) [reply js/settings.js:120]: By deleting that line, you are deleting the "cloak" variable.

**Comment by aeiea** [comment]
fixing

**Review by DapperLS-Bot-Dev — APPROVED** [review]
I totally read the changes, approved.

**Pull request merged**